beacon-chain/rpc/beacon_server.go

```diff
@@ -216,7 +331,6 @@ func (bs *BeaconServer) BlockTreeBySlots(ctx context.Context, req *pb.TreeBlockS
 	}, nil
 }
 
-// TODO(#2307): Refactor for v0.6.
 // nolint
 func (bs *BeaconServer) defaultDataResponse(ctx context.Context, currentHeight *big.Int, eth1FollowDistance int64) (*pbp2p.Eth1Data, error) {
```

Comment:
This name is ambiguous and looks like it should be included somewhere else or removed altogether

Reply:
changed the name